include/cantera/oneD/StFlow.h

```diff
@@ -74,11 +74,15 @@ class StFlow : public Domain1D
     }
 
     //! set the transport manager
-    void setTransport(Transport& trans, bool withSoret = false);
+    void setTransport(Transport& trans, bool withSoret = false, bool withAmbi = false);
     void enableSoret(bool withSoret);
+	void enableAmbi(bool withAmbi);
```

Comment:
I think this function name should be `enableAmbipolar`. Similarly for other functions where just `ambi` is used in the identifier. Also, a docstring should be added for this function.